src/workspace.coffee

```diff
@@ -47,6 +47,8 @@ class Workspace extends Model
       left: new PanelContainer({location: 'left'})
       right: new PanelContainer({location: 'right'})
       bottom: new PanelContainer({location: 'bottom'})
+      topBar: new PanelContainer({location: 'top-bar'})
+      bottomBar: new PanelContainer({location: 'bottom-bar'})
```

Comment:
Naming things is probably the single most difficult thing in software development, hehe. I like `top-most` and `bottom-most` too, but something about those names don't "jive".

What about something simple like `header` and `footer`?